BD-948: Compliance Issues in Flutter-Essos

lekshmi-renuka opened the pull request:
Reason for change: Changes to fix compliance issues
Test Procedure: NA
Risks: low
Signed-off-by: Lekshmi Renuka <Lekshmi_Renuka@comcast.com>

lekshmi-renuka committed:
BD-948: Compliance Issues in Flutter-Essos

Reason for change: Changes to fix compliance issues
Test Procedure: NA
Risks: low
Signed-off-by: Lekshmi Renuka <Lekshmi_Renuka@comcast.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/rdkcentral/Flutter-ESSOS?pullRequest=1) <br/>All committers have signed the CLA.

mhughesacn commented:
Thanks Lekshmi: this is what I wanted, thank you.

rdkcmf-jenkins commented:
## Blackduck scan failure details

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/Flutter-ESSOS/1/rdkcentral/Flutter-ESSOS

* Commit: af4e950a4d3d1896989bf5cb0d486c247de8c550

 **Report detail:** [gist](https://gist.github.com/fa11ca762abe24899ee59bc8481e58b3)

rdkcmf-jenkins commented:
## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: Id work done after scan was enabled.

* Commit: af4e950a4d3d1896989bf5cb0d486c247de8c550

shibupthomas reviewed (dismissed):
No Comments.

lekshmi-renuka committed:
BD-948: Compliance Issues in Flutter-Essos

Reason for change: Changes to fix compliance issues
Test Procedure: NA
Risks: low
Signed-off-by: lekshmi-renuka <lekshmi.m@tataelxsi.co.in>

lekshmi-renuka committed:
BD-948: Compliance Issues in Flutter-Essos

Reason for change: Changes to fix compliance issues
Test Procedure: NA
Risks: low
Signed-off-by: lekshmi-renuka <lekshmi.m@tataelxsi.co.in>
